templates/aws/template-variables.rc

```diff
@@ -0,0 +1,16 @@
+# Kubernetes cluster configuration
+export KUBERNETES_VERSION=v1.30.0
+export CONTROL_PLANE_MACHINE_COUNT=1
+export WORKER_MACHINE_COUNT=1
+
+# AWS region
+export AWS_REGION="eu-central-1"
+
+# AWS machine configuration
+export AWS_CREATE_BASTION=true
+export AWS_PUBLIC_IP=false
+export AWS_CONTROL_PLANE_MACHINE_FLAVOR=t3.large
+export AWS_NODE_MACHINE_FLAVOR=t3.large
+export AWS_SSH_KEY_NAME=default
```

Comment:
same here. we do not want typos in user configs to slip through and have placeholder defaults find their way to cluster templates.